pocs/utils/images/__init__.py

```diff
@@ -272,3 +273,33 @@ def _glob(s):
                     warn('Could not delete file: {!r}'.format(e))
     except Exception as e:
         warn('Problem with cleanup creating timelapse: {!r}'.format(e))
+
+
+def upload_observation_dir(pan_id, dir_name, bucket='panoptes-survey'):
+    """ Upload an observation directory to google cloud storage.
+
+    Note:
+        This requires that the command line utility `gsutil` be installed
+        and that authentication has properly been set up.
+
+    Args:
+        dir_name (str): Full path to observation directory.
+    """
+    assert os.path.exists(dir_name)
+    gsutil = shutil.which('gsutil')
+
+    img_path = '{}/*.fz'.format(dir_name)
+    field_dir = dir_name.split('fields')[-1]
+    remote_path = '{}/{}/'.format(pan_id, field_dir).replace('//', '/')
```

Comment:
I noticed when running the previous script I noticed that there were some double slashes in the linux path. For example:
```python
CommandException: No URLs matched: /var/panoptes/images/fields//HatP30/6575fc/20180127T023054/*.fz
CommandException: 1 file/object could not be transferred.
```
It might be good to use os.join.path rather than string manipulation.